core/domain/suggestion_registry.py

```diff
@@ -1035,7 +1050,7 @@ def pre_accept_validate(self) -> None:
             raise utils.ValidationError(
                 'The skill with the given id doesn\'t exist.')
 
-    # We have ignored [override] here because the signature of this method
+    # Here we use MyPy ignore because the signature of this method
```

Comment:
Why can't we make it match? There should be more explanation.

Reply:
Because in BaseSuggestion it is expected to return a sequence of BaseChange 
https://github.com/oppia/oppia/blob/68bb129c7fe773b989670a59162534b8893789bb/core/domain/suggestion_registry.py#L294
but in this sub-class's method we are not returning anything which causes mypy to throw an error.

Comment is also updated!

Reply:
Fixed!

Reply:
This method is removed!